### PR TITLE
Feature side bar

### PR DESCRIPTION
**Pull request by Airi7911**
3-Practice-01のside-barが完成しました。

### CODE REVIEW

File: src/3-Practice/01/assets/stylesheets/object/components/_sidebar.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.c-sidebar {
+  width: 100%;
+  height: 100%;
+}
+
+@media screen and (max-width: 795px) {
+  .c-sidebar {
+    height: auto;
+  }
+}
```
Comment (kichikawa57) [inline src/3-Practice/01/assets/stylesheets/object/components/_sidebar.css:10]: [must]
Please move to Layout and rename these class name from  `c` to `l`.

File: src/3-Practice/01/assets/stylesheets/object/projects/_index.css [hunk 2]
```diff
@@ -11,9 +11,29 @@
 
 .p-header__image {
   width: 100%;
+  height: 100%;
+  object-fit: cover;
```
Comment (kichikawa57) [inline src/3-Practice/01/assets/stylesheets/object/projects/_index.css:15]: [imo]
Please remove it 🙏

File: src/3-Practice/01/assets/stylesheets/object/components/_sidebar.css (hunk 1 above)
Comment (Airi7911) [reply src/3-Practice/01/assets/stylesheets/object/components/_sidebar.css]: Modified saide bar of css from component to layout.

File: src/3-Practice/01/assets/stylesheets/object/projects/_index.css (hunk 2 above)
Comment (Airi7911) [reply src/3-Practice/01/assets/stylesheets/object/projects/_index.css]: Fixed.

**Review by kichikawa57 — APPROVED** [review]
LGTM

**Pull request merged**